src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -101,6 +101,34 @@ private void testCreateInstructorAction() throws Exception {
                      homePage.getMessageFromResultTable(2));
         homePage.clearInstructorDetailsSingleLineForm();
         
+        ______TS("action success: displayed instructor details are properly HTML-encoded");
+        
+        InstructorAttributes dangerousInstructor = new InstructorAttributes();
+        
+        String shortNameDangerous = "<b>MaliciousInstrúctör</b>";
+        dangerousInstructor.name = "Malicious <script>alert('dangerous');</script>Instrúctör";
+        dangerousInstructor.email = "malicious.instr1<>!@gmail.tmt";
+        String dangerousInstitute = "TEAMMATES Malicious Institute <!@!@!>";
+        String dangerousDemoCourseId = "malicious.instr1___.gma-demo";
+        
+        BackDoor.deleteAccount(TestProperties.TEST_INSTRUCTOR_ACCOUNT);
```

Comment:
It'll be good to repeat this block at the end of the test to clean up the created course and instructor. If you look at the last 3 lines of code, `demoCourseId` and `instructor` are deleted and that was sufficient as all the tests used this same set of IDs. However, now there is this malicious set (`dangerousDemoCourseId` and `dangerousInstructor `) that needs to be cleaned up too.